### PR TITLE
Added recipe for pocketsphinx

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

File: recipes/pocketsphinx/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
```
Comment (jakirkham) [inline recipes/pocketsphinx/meta.yaml]: Please add `toolchain`.

File: recipes/pocketsphinx/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (jakirkham) [inline recipes/pocketsphinx/meta.yaml]: Please add the following.

```
  build:
    features:
      - vc9   # [win and py27]
      - vc10  # [win and py34]
      - vc14  # [win and py35]
```

Comment (jakirkham) [inline recipes/pocketsphinx/meta.yaml]: Please add the following.

```yaml
  features:
    - vc9   # [win and py27]
    - vc10  # [win and py34]
    - vc14  # [win and py35]
```

**Comment by jakirkham** [comment]
Not sure what to do on Windows quite yet. It builds fine on Python 3.5 (Visual Studio 2015). It fails on Python 2.7 (Visual Studio 2008), which is [expected]( https://github.com/cmusphinx/pocketsphinx/blob/ec89abd9d2e976019338ecae7d6cafeecdb95eb2/README#L73 ). However, it fails on Python 3.4 (Visual Studio 2010), which appears it is suppose to build correctly on. Any thoughts on this, @patricksnape?

**Comment by patricksnape** [comment]
Visual Studio 2010 still has terrible C99 support - so I would be surprised if that C-code was expected to build. You could always try patching it to build it as C++? This may also cause it to work on VS 2008.

**Comment by patricksnape** [comment]
So [this](https://github.com/cmusphinx/pocketsphinx/blob/a60982363101704eca342e7e0920754090cd49b1/src/libpocketsphinx/kws_search.c#L462) is a C89 error. But it's fixed in master. So maybe just either patch the variable declaration to be at the top of the function or apply [this commit](https://github.com/cmusphinx/pocketsphinx/commit/c8889b3ecb2240da81566c69447683bf9880b2da) as a patch?

**Comment by jakirkham** [comment]
Thanks for helping narrow that down, @patricksnape. 😄

**Comment by patricksnape** [comment]
I love Visual Studio 😆 😭

**Comment by msarahan** [comment]
That's something like "stare into the eyes of the beast long enough and you become it," right?

**Comment by patricksnape** [comment]
[Yes.](https://en.wikipedia.org/wiki/Stockholm_syndrome)

**Comment by jakirkham** [comment]
I'm guessing you are referring to that quote from Nietzsche...

> He who fights with monsters should be careful lest he thereby become a monster. And if thou gaze long into an abyss, the abyss will also gaze into thee.

File: recipes/pocketsphinx/meta.yaml (hunk 2 above)
Comment (jakirkham) [inline recipes/pocketsphinx/meta.yaml]: Done below.

**Comment by jakirkham** [comment]
How do you feel about adding that patch, @pmlandwehr ?

**Pull request closed without merging**